knowledge-base/how-to-expand-document-map-tree-by-default-in-html5-viewers.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Expand Document Map tree by default in Html5 Report Viewers
+description: How to expand Document Map tree initially in the Html5 Viewers
+type: how-to
+page_title: Make Bookmark tree expanded by default in Html5 Viewers
+slug: how-to-expand-document-map-tree-by-default-in-html5-viewers
+position: 
+tags: 
+ticketid: 1398733
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Reporting</td>
+	</tr>
+</table>
+
+
+## Description
+You may want to have the Bookmark (Document Map) tree expanded by default. Currently, the user has to click on the tree node and expand it after the page is rendered.
+
+## Solution
+The _DocumentMap_ area is a [Kendo UI TreeView](https://docs.telerik.com/kendo-ui/api/javascript/ui/treeview) widget that is loaded on request with nested nodes. Expanding initially all nodes is not supported out of the box.   
```

Comment:
I did not understand what is the meaning of "is loaded on request with nested nodes".

Reply:
modified